### PR TITLE
Camera - Allow TorchControl usage in shared mode documentation

### PR DESCRIPTION
**Pull request by kasumman**
Documenting change that was serviced to older versions of Windows to OEMs/admins to configure camera pipeline to allow TorchControl from shared MediaCapture Sessions to support certain scenarios.

### CODE REVIEW

**Review by saredd — APPROVED** [review]
:shipit:

File: windows-driver-docs-pr/stream/oem-guidance-on-registry-keys-for-shared-torch-usage.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+## Remarks
+This functionality is available starting in
+| Windows 10 version | KB article |
+|--|--|
+| 19041.906 and 19042.906 | [KB8000842](https://support.microsoft.com/topic/march-29-2021-kb5000842-os-builds-19041-906-and-19042-906-preview-1a58a276-6a0a-47a5-aa7d-97af2d10b16d)|
+| 18363.1533 | [KB5001396](https://support.microsoft.com/topic/april-22-2021-kb5001396-os-build-18363-1533-preview-e67788f0-4e70-4f9b-9c5e-ff977310eeea)|
+| 17763.1911| [KB5001384](https://support.microsoft.com/topic/april-22-2021-kb5001384-os-build-17763-1911-preview-e471f445-59be-42cb-8c57-5db644cbc698)|
+- By default this registry value does not exist. 
+- TorchControl does not have an event mechanism to notify state changes thus by controlling the torch from multiple applications may cause the torch being in different state than one application thinks if the state was change by another application.
```
Comment (NaveenThum) [inline windows-driver-docs-pr/stream/oem-guidance-on-registry-keys-for-shared-torch-usage.md:44]: Any other downside to using this registry key? Something that we can tell as to why this is not the default behavior?

Comment (kasumman) [reply windows-driver-docs-pr/stream/oem-guidance-on-registry-keys-for-shared-torch-usage.md:44]: Top of my head that's the main problem thus not default behavior.

File: windows-driver-docs-pr/stream/oem-guidance-on-registry-keys-for-shared-torch-usage.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
```
Comment (saredd) [inline windows-driver-docs-pr/stream/oem-guidance-on-registry-keys-for-shared-torch-usage.md:44]: >change [](http://example.com/codeflow?start=219&length=6)

nit: changed

**Pull request merged**